src/python/national_voter_file/us_states/nc/transformer.py

```diff
@@ -83,7 +83,7 @@ def extract_do_not_call_status(self, input_dict):
 
     def extract_gender(self, input_dict):
         gender = input_dict['gender_code']
-        if len(gender) == 0:
+        if len(gender) == 0 or gender == ' ':
```

Comment:
ah. 🤔 I'm flexible. if not are m and f and there are a few Us, I think so. if the file has none specified for any/most rows, then I think we should go with None. I hope I'm making sense